astropy/samp/hub.py

```diff
@@ -827,9 +827,7 @@ def _declare_metadata(self, private_key, metadata):
         self._update_last_activity_time(private_key)
         if private_key in self._private_keys:
             log.debug(
-                "declare_metadata: private-key = {} metadata = {}".format(
-                    private_key, str(metadata)
-                )
+                f"declare_metadata: private-key = {private_key} metadata = {str(metadata)}"
```

Comment:
This can be shortened:
```suggestion
                f"declare_metadata: private-key = {private_key} {metadata = !s}"
```

Reply:
Even shorter:

```suggestion
                f"declare_metadata: private-key = {private_key} {metadata = }"
```